### PR TITLE
Task/770  add application assignment in user edit has been completed

### PR DESCRIPTION
**Pull request by jananikuruntham**

### CODE REVIEW

File: front/src/components/User/EditUserModal.tsx [hunk 1]
```diff
@@ -39,24 +63,130 @@ interface EditModalProps {
...
+    ICreateListProps[]
+  >([]);
+  const [saveAlert, setSaveAlert] = useState<AlertState | null>(null);
+  const [applications, setApplications] = React.useState<Application[]>([]);
+  const [id, setId] = useState<number | undefined>(params.id);
+  const [existingCheckboxes, setExistingCheckboxes] = useState<
+    ICreateListProps[]
+  >([]);
+  const [value, setValue] = React.useState(0);
+  const [invalidEmail, setInvalidEmail] = useState("");
```
Comment (vsaravanakumar) [inline front/src/components/User/EditUserModal.tsx:182]: Remove unused variables if not need

File: front/src/components/User/EditUserModal.tsx [hunk 2]
```diff
@@ -39,24 +63,130 @@ interface EditModalProps {
...
+  mobile: string;
+  id: number;
+}
+
+const CustomTab = styled(Tab)(({ theme }) => ({
+  textTransform: "none",
+}));
+
+const selectStyles: StylesConfig = {
+  container: (base) => ({
```
Comment (vsaravanakumar) [inline front/src/components/User/EditUserModal.tsx:121]: Solve eslint error

File: front/src/components/User/EditUserModal.tsx [hunk 3]
```diff
@@ -7,15 +7,35 @@ import {
...
 import SaveIcon from "@mui/icons-material/Save";
 import CloseIcon from "@mui/icons-material/Close";
+import SearchIcon from "@mui/icons-material/Search";
 import { RowData } from "./UserList";
-import { MenuItem } from "@mui/material";
 import { RolesApi } from "@/services/api/RolesApi";
-
+import { UserApi } from "@/services/api/UserApi";
+import PropTypes from "prop-types";
+import { StylesConfig } from "react-select";
```
Comment (vsaravanakumar) [inline front/src/components/User/EditUserModal.tsx:38]: Solve eslint error

File: front/src/components/User/EditUserModal.tsx [hunk 4]
```diff
@@ -39,24 +63,130 @@ interface EditModalProps {
...
+  );
+}
+
+CustomTabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.number.isRequired,
+  value: PropTypes.number.isRequired,
+};
+
+function a11yProps(index: number) {
```
Comment (vsaravanakumar) [inline front/src/components/User/EditUserModal.tsx:149]: Change the function name

File: front/src/components/User/EditUserModal.tsx (hunk 1 above)
Comment (jananikuruntham) [reply front/src/components/User/EditUserModal.tsx:182]: Removed unused variables

File: front/src/components/User/EditUserModal.tsx (hunk 2 above)
Comment (jananikuruntham) [reply front/src/components/User/EditUserModal.tsx:121]: solved eslint error

File: front/src/components/User/EditUserModal.tsx (hunk 4 above)
Comment (jananikuruntham) [reply front/src/components/User/EditUserModal.tsx:149]: removed edituser modal

**Pull request merged**